Bounce/dist/metadata.json

```diff
@@ -10,6 +10,7 @@
     "service": "Bounce",
     "description": {
         "en": "Bounce Radio is a 24/7 international hit music station that provides hit music to everyone. Made by gamers for gamers.",
+        "fr": "Bounce Radio est une station de musique à succès internationale 24/7 qui fournit de la musique à succès à tout le monde. Fabriqué par des joueurs pour des joueurs.",
         "nl": "Bounce Radio is een 24/7 internationaal hitmuziekstation dat iedereen hitmuziek biedt. Gemaakt door gamers voor gamers.",
```

Comment:
please remove the comma behind the last translation.

Reply:
Removed :)